tt1-blocks/functions.php

```diff
@@ -73,19 +59,3 @@ function tt1_blocks_enqueue_styles() {
 	wp_enqueue_style( 'tt1-blocks-blocks', get_template_directory_uri() . '/assets/css/blocks.css', array(), wp_get_theme()->get( 'Version' ) );
 }
 add_action( 'wp_enqueue_scripts', 'tt1_blocks_enqueue_styles' );
-
-/**
- * Enqueue block editor script.
- *
- * @since 0.1
- *
- * @return void
- */
-function tt1_blocks_block_editor_script() {
-
-	wp_enqueue_script( 'tt1-blocks-unregister-block-style', get_theme_file_uri( '/assets/js/unregister-block-style.js' ), array( 'wp-blocks', 'wp-dom' ), wp_get_theme()->get( 'Version' ), true );
```

Comment:
I'm not sure why wanted to do this.

Reply:
because we were forcing the width of the main separator via css and both were the same (default and wide were both 100%) and that was redundant. This needs to go